server/src/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -4695,6 +4699,22 @@ private boolean checkIfHostIsDedicated(HostVO host) {
         }
     }
 
+    /**
+     * Checks whether the host is preparing for maintenance mode or not. If not, throws exception with the VM id and operation performed on it
```

Comment:
comment states the reverse of the intended "If not, throws exception..." should be "If so, throws .."
also as you are making javadoc (not needed for a private method, i'd say) best to add a @throws tag as this is the most important functionality of this method.